xcode/Subconscious/Shared/Components/Detail/DetailStackView.swift

```diff
@@ -119,7 +119,7 @@ enum DetailStackAction: Hashable {
     case pushDetail(MemoDetailDescription)
 
     case requestOurProfileDetail
-    case pushOurProfileDetail(UserProfile)
```

Comment:
Since it's cheap to retrieve UserProfile's now I think we should stop passing them around. There was a point where this cut down on load times but not anymore.

Reply:
Nice. I like the idea of cutting down on caches that could get stale. SQLite is our cache, and it should be synchronized in lock-step with Noosphere syncs, so it should have the most up-to-date information, yet be fast to read.